src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,60 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <>
+    <div
+      className="card"
+      data-cy="Movie"
+      key={movie.imdbId}
+    >
```

Comment:
use samntic tag and move key in the movieList component where you call MovieCard